### PR TITLE
feat: add swan images and details

### PR DESCRIPTION
**Pull request by bsatrom**
Getting this all set to go. This PR merges into our fork, and once we put a PR into the main CircuitPython repo for Swan support, I'll put in a PR from our fork to the main.

### CODE REVIEW

File: _board/swan.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  - Battery Charging
+  - Breadboard-Friendly
+---
+
+Swan is a low-cost embeddable STM32L4-based microcontroller designed to accelerate the development and deployment of battery-powered solutions. It is especially useful for applications requiring large memory or a high degree of I/O expandability at an affordable cost, such as edge inferencing and remote monitoring.
+
+Uniquely for Feather-compatible boards, Swan is designed to satisfy developers’ needs that span from early prototyping through high-volume deployment. Developers may begin to use Swan in conjunction with Adafruit’s myriad sensors and FeatherWing-compatible carriers. Due to its novel design, for high-volume deployment the low-cost Swan can also be soldered directly to a parent PCB integrating those sensors, utilizing the full range of Swan’s I/O capabilities.
+
+The board has three independent power options – USB, Battery, or Line power – and provides a software-switchable 2 Amp regulator for powering external sensors. When operating in its low-power operating mode, the entire Swan board commonly draws only about 8uA while retaining all of its memory, making it quite suitable for battery-powered devices.
```
Comment (rdlauer) [inline _board/swan.md:20]: I wouldn't capitalize battery and line

Comment (rdlauer) [reply _board/swan.md:20]: also, I don't like "commonly draws only about"..."commonly" = maybe it does! "about" = somewhere in this range. I'd prefer we just said "draws ~8uA"

File: _board/swan.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+board_id: "swan"
+title: "Swan Download"
+name: "Swan"
+manufacturer: "Blues Wireless"
+board_url: "https://blues.io/products/swan"
+board_image: "swan.jpg"
+date_added: 2021-9-29
+features:
+  - Feather-Compatible
+  - Battery Charging
```
Comment (rdlauer) [inline _board/swan.md:12]: not sure about "Battery Charging" and "Breadboard-Friendly" as features. not sure how prominent these are on the listing (or if this is for SEO), but maybe "55 GPIO" and "Three independent power options" would be better?

**Review by rdlauer — APPROVED** [review]
approved, just made a few comments too

File: _board/swan.md (hunk 2 above)
Comment (bsatrom) [reply _board/swan.md:12]: These are limited by Adafruit in terms of what I can select. 🤷

**Pull request merged**